rest_api/network_apis/route-route-openshift-io-v1.adoc

```diff
@@ -222,7 +221,7 @@ Required::
 | `string`
 | insecureEdgeTerminationPolicy indicates the desired behavior for insecure connections to a route. While each router may make its own decisions on which ports to expose, this is normally port 80.
 
-* Allow - traffic is sent to the server on the insecure port (default) * Disable - no traffic is allowed on the insecure port. * Redirect - clients are redirected to the secure port.
+* Allow - allows insecure connections for an edge-terminated route. * None - disables insecure connections for an edge-terminated route. * Redirect - redirects insecure connections for an edge-terminated route.
```

Comment:
Shouldn't this be a bulleted list?

Reply:
I am afraid if I use bulleted lists for this column the other content wouldn't be parallel? This column could stand out?

Reply:
okay. Bullets are not rendering in the preview/table. so reverting back

Reply:
```suggestion
* Allow - allows insecure connections for an edge-terminated route. * None - disables insecure connections for an edge-terminated route. * Redirect - redirects insecure connections for an edge-terminated route.
```

If you add the `a` macro before the cell marker `|`, the contents of the cell are interpreted as regular asciidoc format. You can get your unordered list this way.